compiler/src/latex_printer.ml

```diff
@@ -235,6 +235,19 @@ let pp_args fmt (sty, xs) =
     pp_sto_ty sty
     (pp_list " " pp_var) xs
 
+  let pp_decl fmt (x:vardecl )= 
```

Comment:
You may take a located `vardecl` here.

Reply:
I was doing L.unloc before calling pp_decl, I change it so it is done in the function